service/analysis.gradle

```diff
@@ -43,6 +43,7 @@ test {
 }
 
 task verifyPacts(type: Test) {
+	outputs.upToDateWhen { false } // always run this task even if it's up to date
 	useJUnitPlatform {
 		includeTags 'Pact'
 	}
```

Comment:
```suggestion
	}
	testLogging {
		showStandardStreams = true
	}
```

Reply:
By default, `JUnit` outputs the log to `xml` file. The config `showStandardStreams = true` will direct `logback` to the console so that the logs will show up in the verification workflow.